lib/plugins/processor/asset.ts

```diff
@@ -49,7 +49,7 @@ function processPage(ctx, file) {
     file.stat(),
     file.read()
   ]).spread((stats, content) => {
-    const data = yfm(content);
```

Comment:
IMHO you should update type for `hexo-front-matter` instead.

Reply:
because `hexo-front-matter` doesnt have optional argument for options, `hexo` must input empty option for successful build

![image](https://user-images.githubusercontent.com/12471057/233777991-942787de-34f4-4ee1-b186-fcc445759596.png)

Reply:
should i modify `hexo-front-matter` package too ?

Reply:
i using monorepo to sync between `hexo-front-matter` and `warehouse` take look at https://github.com/dimaslanjaka/hexo/tree/monorepo

Reply:
restored original imports https://github.com/dimaslanjaka/hexo/commit/6d5bb6442d639dc96428e916f3f12dd5bc80cb64